src/libs/antares/study/binding_constraint/BindingConstraint.cpp

```diff
@@ -545,6 +545,10 @@ double BindingConstraint::weight(const AreaLink* lnk) const
 
 double BindingConstraint::weight(const ThermalCluster* cluster) const
 {
+    if (!cluster->isEnabled())
```

Comment:
Add a comment : "this method should only be used in the GUI, it doesn't handle inactive clusters well"

```suggestion
```